packages/client/lib/net/protocol/snapprotocol.ts

```diff
@@ -124,11 +154,12 @@ export class SnapProtocol extends Protocol {
       name: 'GetStorageRanges',
       code: 0x02,
       response: 0x03,
+      // [reqID: P, rootHash: B_32, accountHashes: [B_32], startingHash: B, limitHash: B, responseBytes: P]
       encode: ({ reqId, root, accounts, origin, limit, bytes }: GetStorageRangesOpts) => {
         return [
           bigIntToBuffer(reqId ?? ++this.nextReqId),
-          root,
-          accounts,
+          setLengthLeft(root, 32),
```

Comment:
I am honestly a bit surprised that the spec marks this as a `B_32`. We could save some bytes if the hash has leading zeros, why not shave those off?